docs/references/api/tables_views.rst

```diff
@@ -193,7 +194,16 @@ The :code:`fts` filter mentioned above has a number of options to support flexib
 
   curl "http://localhost:3000/tsearch?my_tsv=not.wfts(french).amusant"
 
-Using `websearch_to_tsquery` requires PostgreSQL of version at least 11.0 and will raise an error in earlier versions of the database.
+For other column types like ``text`` or ``json``, you may want to explicitly convert these columns using ``to_tsvector()``, specially when a language is specified.
+To do so, use the ``totsv`` modifier:
+
+.. code-block:: bash
+
+  curl "http://localhost:3000/tsearch?text_column=totsv.fts(french).amusant"
```

Comment:
Hm, at a glance this looks too ad hoc and leaking internal details through the API.

Could we do the casting automatically with the schema cache approach mentioned on https://github.com/PostgREST/postgrest/issues/2255#issuecomment-1111220493?

Reply:
We could. But we wouldn't know if a computed column ([created in the way we mention in the docs](https://docs.postgrest.org/en/v12/references/api/computed_fields.html#computed-fields)) is of `tsvector` type or not.

Hmm... to solve this we could apply `to_tsvector('language',column)` only to those we know are `text` or json types and let PostgREST do the implicit casting to the rest. Still if other computed column with text or json types are present they wouldn't be using the `to_tsvector` and the problem would persist for those.

Reply:
> But we wouldn't know if a computed column ([created in the way we mention in the docs](https://docs.postgrest.org/en/v12/references/api/computed_fields.html#computed-fields)) is of tsvector type or not.

Why? Care to elaborate?

I haven't seen computed fields discussed on the issue, are they really relevant for this use case?

Reply:
> Why? Care to elaborate?

We cannot detect the computed field because it's not a column of the table, so it won't be present in the schema cache. For example, what would you suggest in those cases? 
  * Use `to_tsvector()` in the computed field anyways, but if it returns a `tsvector` it would err, or
  * Don't add anything and let the issue happen for computed fields only.

> I haven't seen computed fields discussed on the issue, are they really relevant for this use case?

Ah no, they weren't mentioned, just trying to cover all possibilities here. e. g. From the example in the docs, using `?full_name=totsv.fts(simple).and` would work, but using the schema cache (if we don't do anything for computed columns), it wouldn't.

Reply:
TBH I also like the approach of auto-detecting the type, but since it didn't cover everything I didn't go for it. We could implement that and wait to see if there are actually problems with users and computed fields or, better yet, document it in some way (since they're already using computed fields, creating a `to_tsvector` one should be the solution). WDYT?

Reply:
> Use to_tsvector() in the computed field anyways, but if it returns a tsvector it would err, or

That sounds fine to me for now. Later on we can make it a more clear error if we add computed fields to the schema cache.

Reply:
Hmmm... this would break responses for those who are already using `tsvector` computed cols. I'd prefer to not do anything for those cases. Maybe we need @wolfgangwalther to break our stalemate here?

Reply:
Cool, I'll keep the computed columns for later then. Tracked here #956